src/Tizen.NUI.Scene3D/src/public/ModelComponents/ModelPrimitive.cs

```diff
@@ -142,8 +142,24 @@ private void SetGeometry(Geometry geometry)
         private Geometry GetGeometry()
         {
             IntPtr cPtr = Interop.ModelPrimitive.GetGeometry(SwigCPtr);
+            Geometry ret = Registry.GetManagedBaseHandleFromNativePtr(cPtr) as Geometry;
+            if (ret == null)
+            {
+                // Store the value of PositionUsesAnchorPoint from dali object (Since View object automatically change PositionUsesPivotPoint value as false, we need to keep value.)
```

Comment:
Looks not matched comments. remove this sentense is more clear I think.

Reply:
Thank you I removed them.